workflow/scripts/featurecounts_deseq2.R

```diff
@@ -285,16 +285,30 @@ if (file.exists(ResultsFile) == FALSE) {
 
                 ## WRITE RESULTS FILE
                 if (MIN_FDR == 0.01) {
-                    CompResultsFile <- snakemake@output[["FDR_1_perc_res"]]
-                    CompBEDFile <- snakemake@output[["FDR_1_perc_bed"]]
-                    MAplotFile <- snakemake@output[["plot_FDR_1_perc_MA"]]  # AVI: added to create separate pdf files
-                    VolcanoPlotFile <- snakemake@output[["plot_FDR_1_perc_volcano"]]  # AVI: added to create separate pdf files
+                    # AVI: dynamically generated file name extensions added
+                    dirs <- c(snakemake@output[["FDR_1_perc_res"]],snakemake@output[["FDR_1_perc_bed"]],snakemake@output[["plot_FDR_1_perc_MA"]],
+                              snakemake@output[["plot_FDR_1_perc_volcano"]],snakemake@output[["FDR_5_perc_res"]],snakemake@output[["FDR_5_perc_bed"]],
+                              snakemake@output[["plot_FDR_5_perc_MA"]],snakemake@output[["plot_FDR_5_perc_volcano"]],snakemake@output[["plot_sample_corr_heatmap"]],
+                              snakemake@output[["plot_scatter"]])
```

Comment:
```suggestion
                    dirs <- c(
                        snakemake@output[["FDR_1_perc_res"]],
                        snakemake@output[["FDR_1_perc_bed"]],
                        snakemake@output[["plot_FDR_1_perc_MA"]],
                        snakemake@output[["plot_FDR_1_perc_volcano"]],
                        snakemake@output[["FDR_5_perc_res"]],
                        snakemake@output[["FDR_5_perc_bed"]],
                        snakemake@output[["plot_FDR_5_perc_MA"]],
                        snakemake@output[["plot_FDR_5_perc_volcano"]],
                        snakemake@output[["plot_sample_corr_heatmap"]],
                        snakemake@output[["plot_scatter"]]
                    )
```
I think this makes it a bit easier to get an overview of all the outputs.